src/middlewared/middlewared/alembic/versions/22.12/2022-04-01_16-26_merge.py

```diff
@@ -0,0 +1,21 @@
+"""merge
+
+Revision ID: e391384840d0
+Revises: 19eb67dcdee2
+Create Date: 2022-04-01 16:26:31.521475+00:00
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'e391384840d0'
+down_revision = '19eb67dcdee2'
```

Comment:
It looks like it is not a merge migration, merge migrations have > 1 parents.

Reply:
Should be fixed